source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,140 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon a
```

Comment:
> Because a query can potentially match large sets of documents

Now that this page is also describing `first` and `into`, this language should be reconsidered.